### PR TITLE
Handle errors in switching to mobile webviews

### PR DESCRIPTION
**Pull request by rajukamal**

### CODE REVIEW

File: utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java [hunk 1]
```diff
@@ -75,7 +78,16 @@ final AppiumDriver switchToWebView(String title) {
...
       if (!contextHandle.equals(NATIVE_CONTEXT_HANDLE)) {
-        AppiumDriver newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
+        AppiumDriver newDriver;
+        try {
+          newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
+        } catch (WebDriverException e) {
+          warning("Context switch to webview '" + contextHandle + "' failed. Error : " + e.getMessage());
+          continue;
+        }
+        if (newDriver == null) {
```
Comment (helenren) [inline utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java:88]: would suggest to use if (newDriver != null) {// do title check }

Comment (rajukamal) [reply utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java]: updated

**Review by helenren — APPROVED** [review]

**Comment by helenren** [comment]
@lizaiv77 here is a fix for mobile specifically. Please have a check. Thanks.

File: utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java [hunk 2]
```diff
@@ -75,10 +78,18 @@ final AppiumDriver switchToWebView(String title) {
...
       if (!contextHandle.equals(NATIVE_CONTEXT_HANDLE)) {
-        AppiumDriver newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
-        String newTitle = newDriver.getTitle();
-        if (!newTitle.isEmpty() && newTitle.equalsIgnoreCase(title)) {
-          return newDriver;
+        AppiumDriver newDriver;
+        try {
+          newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
+        } catch (WebDriverException e) {
+          warning("Context switch to webview '" + contextHandle + "' failed. Error : " + e.getMessage());
```
Comment (lizaiv77) [inline utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java:85]: nit: pls use `String.format("...", contextHandle, e.getMessage())` for consistency with other code and remove space before semicolon

**Review by lizaiv77 — COMMENTED** [review]
minor comment

File: utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java (hunk 2 above)
Comment (rajukamal) [reply utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java]: @lizaiv77 have updated.

**Review by lizaiv77 — APPROVED** [review]

**Pull request merged**